app/admin/RTMediaUploadTermsAdmin.php

```diff
@@ -0,0 +1,209 @@
+<?php
+/**
+ * RTMediaUploadTermsAdmin.php
+ * This file sets the admin settings and handle feature of Upload Terms at admin side.
+ *
+ * @package rtMedia
+ * @author  Malav Vasita <malav.vasita@rtcamp.com>
+ */
+
+/**
+ *  Class for Admin settings regarding Upload Terms
+ */
+include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+if ( ! class_exists( 'RTMediaUploadTermsAdmin' ) && ! is_plugin_active( 'rtmedia-upload-terms/index.php' ) ) {
```

Comment:
An inline comment explaining the need of `if` condition would be appreciated. Especially for this part `! is_plugin_active( 'rtmedia-upload-terms/index.php' )`.

Reply:
@malavasita  If you want to execute this code only when `rtmedia-upload-terms`  plugin is activated then the `if` condition won't allow you to do so. I can say this by looking at this `!` in `if` condition.
Correct me if I am wrong.

Reply:
Yes. we want to be accessible to this class if the old plugin is deactivated.
That is why we have put `!` ahead of `if`.

If the old plugin is deactivated then `is_plugin_active( 'rtmedia-upload-terms/index.php' )` will return false. in that case, we want to call this function. that is why `!` is there. that will make `false` in `true` and then we will be accessible to this class.